torch/csrc/jit/codegen/cuda/lower_alias_memory.cpp

```diff
@@ -86,7 +86,7 @@ class AllocateReuseModifier {
       const auto input_alloc = findCompatibleInputAllocate(
           tv->dtype(), SymbolicSizePrinter::printSize(output_alloc), def);
 
-      if (input_alloc != nullptr) {
+      if (input_alloc != nullptr && output_alloc->vectorSize() == nullptr) {
```

Comment:
Why can't vectorized allocations be aliased like other allocations?

Reply:
It doesn't generate the vectorized instruction in sass and fails correctness.